nat-lab/tests/test_vpn.py

```diff
@@ -15,27 +15,21 @@
 
 
 async def _connect_vpn(
-    connection: Connection,
-    vpn_connection: Connection,
+    client_conn: Connection,
     client: Client,
-    client_meshnet_ip: str,
     wg_server: dict,
 ) -> None:
     await client.connect_to_vpn(
         wg_server["ipv4"], wg_server["port"], wg_server["public_key"]
     )
 
-    async with Ping(connection, config.PHOTO_ALBUM_IP).run() as ping:
+    async with Ping(client_conn, config.PHOTO_ALBUM_IP).run() as ping:
         await testing.wait_long(ping.wait_for_next_ping())
 
-    async with Ping(vpn_connection, client_meshnet_ip).run() as ping:
```

Comment:
this ping has to stay here, it was added for a reason. See https://github.com/NordSecurity/libtelio/pull/76

Reply:
I've restored it and placed it as an optional parameter so nordlynx servers wouldn't be pinged. I didn't investigate further just talked with @Jauler some time ago about it and he mentioned that we can skip the pinging so doing exactly that just for my introduced server.